tests/conftest.py

```diff
@@ -208,3 +208,11 @@ def pytest_collection_modifyitems(items):
             # has `@RunIf(slow=True)`
             if marker.name == "skipif" and marker.kwargs.get("slow")
         ]
+    elif os.getenv("PL_RUN_IPU_TESTS", "0") == "1":
```

Comment:
can we rather have some internal device identification, because in most cases you won't have both IPU/TPU/GPU in one machine... and it can be easy to forget in the environment to enable it...

Reply:
Could be a nice improvement! Could we merge this first and introduce the logic for all accelerator-specific (`{g,h,t,i}pu`) tests later in another PR?